projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -1665,12 +1665,11 @@ public function get_media_item( $media_id ) {
 	 * @return object|WP_Error Media item data, or WP_Error.
 	 */
 	public function get_media_item_v1_1( $media_id, $media_item = null, $file = null ) {
-
 		if ( ! $media_item ) {
 			$media_item = get_post( $media_id );
 		}
 
-		if ( ! $media_item || is_wp_error( $media_item ) ) {
+		if ( ! $media_item || is_wp_error( $media_item || ! ( $media_item instanceof WP_Post ) ) ) {
```

Comment:
Should condition go like this? The `instanceof` will return bool so checking it for `is_wp_error` will always return false?
```suggestion
		if ( ! $media_item || is_wp_error( $media_item )  || ! ( $media_item instanceof WP_Post ) ) ) {
```

Reply:
Yes, thank you! I mis-placed the closing bracket somehow. Given that was the only thing that I was unable to properly test, that's a good catch.
I added the change via a separate commit - adc1139ac99ce2f87e54f4affa1c1b6ae84ae3f3 - as there is an extra space in the example above.